src/DocumentFormat.OpenXml/Packaging/Model3DReferenceRelationshipPart.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+using DocumentFormat.OpenXml.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO.Packaging;
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    /// <summary>
+    /// Defines the Model3DReferenceRelationshipPart
+    /// </summary>
+    public partial class Model3DReferenceRelationshipPart : OpenXmlPart, IFixedContentTypePart
```

Comment:
nit: no need for the base types here

Reply:
Do I need the /// <summary> block or can I also use /// <inheritdoc/> on it like below?

Reply:
Nope, actually neither are needed. It's a partial class, so the original part will have the comments. I'm not certain how comments for partial classes work - they may be combined, or just the first one is used. Either way, you can just remove these.